ci/unit/test_post_always.sh

```diff
@@ -25,10 +22,36 @@ mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
 # shellcheck disable=SC2029
 ssh "$SSH_KEY_ARGS" jenkins@"$NODE" \
-  "STAGE_NAME='$STAGE_NAME'         \
-   DAOS_BASE=$DAOS_BASE             \
+  "DAOS_BASE=$DAOS_BASE             \
    $(cat "$mydir/test_post_always_node.sh")"
 
+dnt_logs=false
+case $STAGE_NAME in
+    *Bullseye*)
+	test_log_dir="covc_test_logs/"
+	;;
+    *memcheck*)
+	test_log_dir="unit_test_memcheck_logs/"
+	;;
+    *Unit*)
+	test_log_dir="unit_test_logs/"
+	;;
+    *NLT*)
+	test_log_dir="nlt_logs/"
+	dnt_logs=true
+	;;
+esac
+
+rm -rf $test_log_dir
+mkdir $test_log_dir
+
+if $dnt_logs
+then
+    rsync -av -z -e "ssh $SSH_KEY_ARGS" jenkins@"$NODE":/tmp/dnt*.log $test_log_dir
+else
+    rsync -av -z -e "ssh $SSH_KEY_ARGS" jenkins@"$NODE":/tmp/daos*.log $test_log_dir
+fi
```

Comment:
Very minor nit and certainly nothing I would -1 for, but just something to think about...

```suggestion
rsync_logs() {
    local source="$1"
    local dest="$2"
    
    rsync -av -z -e "ssh $SSH_KEY_ARGS" jenkins@"$NODE":"$source" "$dest"
}    
    
if $dnt_logs; then
    rsync_logs /tmp/dnt\*.log "$test_log_dir"
else
    rsync_logs /tmp/daos\*.log "$test_log_dir"
fi
```

Just to dry out the duplication of the `rsync` command.